app/backend/core/authentication.py

```diff
@@ -57,11 +59,14 @@ def __init__(
                 client_credential=server_app_secret,
                 token_cache=PersistedTokenCache(persistence),
             )
+        else:
+            self.require_access_control = False
```

Comment:
I also see a warning in the logs for line 56, so I've filed https://github.com/AzureAD/microsoft-authentication-library-for-python/issues/629 to get MSAL's recommendation for how to avoid that warning.

Reply:
got it, looks like this is going to be fixed in the msal library?

Reply:
Yep they merged a fix